packages/views.vite-plugin/README.md

```diff
@@ -0,0 +1,163 @@
+# @osdk/views-manifest-vite-plugin
+
+This is a vite plugin that will automatically discover `*.config.(j|t)s` files that are imported into entrypoint files and generates a `.palantir/view-config.json` file for Foundry to read from. This allows developers to write out their configuration for custom views in Foundry in a single, type-safe file and not worry about generating a manifest file needed for publishing new versions of the view within Foundry.
```

Comment:
We are fairly confident that nothing else uses the `*.config.js` format right? Or at least in the fact that only considering `.config.js` imports from the entrypoint avoids some set of potential collisions

Reply:
if we wanted to be super safe we could probably to `.foundry.config.js` ..?

Reply:
actually i realized it shouldn't matter - esp if we add validation on the parsed config object, we can ignore any other `.config.js` file that doesn't fit what we're looking for